cmd/plank_mock.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import "github.com/armory/plank/v3"
+
+var pipelineID string
+
+type PlankMock struct {
+}
+
+func (p PlankMock) GetApplication(string string) (*plank.Application, error){
+	return &plank.Application{
+		Name:          string,
+		Email:         "",
```

Comment:
If you don't provide the fields they'll default to these values anyway 🤷

Reply:
Deleted them